docs/source/routing/performance/caching/entity.mdx

```diff
@@ -135,6 +213,57 @@ This entry contains an object with the `all` field to affect all subgraph reques
 
 ### Entity cache invalidation
 
+```mermaid
+---
+title: Invalidating an Entity Cache
+---
+
+flowchart RL
+	subgraph QueryResponse["Cache invalidation POST"]
+		n1["{
+			&emsp;&emsp;&quot;kind&quot;: &quot;subgraph&quot;,
+			&emsp;&emsp;&quot;subgraph&quot;: &quot;price&quot;,
+			&emsp;&emsp;&quot;type&quot;: &quot;Price&quot;,
+			&emsp;&emsp;&quot;key&quot;: {
+			&emsp;&emsp;&emsp;&emsp;&quot;id&quot;: &quot;101&quot;
+			&emsp;&emsp;}
+		}"]
+    end
+
+	subgraph Subgraphs["Subgraphs"]
+		Purchases["Purchases"]
+		Inventory["Inventory"]
+		Price["Price"]
+	end
+
+	subgraph PriceQueryFragment["Price Query Fragment (e.g. TTL 2200)"]
+		n2[" ̶{̶
+    	&emsp;&emsp;&quot; ̶p̶r̶i̶c̶e̶&quot;:  ̶{̶
+        &emsp;&emsp;&emsp;&emsp;&quot; ̶i̶d̶&quot;:  ̶1̶0̶1̶,
+        &emsp;&emsp;&emsp;&emsp;&quot; ̶p̶r̶o̶d̶u̶c̶t̶_̶i̶d̶&quot;:  ̶1̶2̶,
+        &emsp;&emsp;&emsp;&emsp;&quot; ̶a̶m̶o̶u̶n̶t̶&quot;:  ̶1̶5̶0̶0̶,
+        &emsp;&emsp;&emsp;&emsp;&quot;̶c̶u̶r̶r̶e̶n̶c̶y̶_̶c̶o̶d̶e̶&quot;: &quot; ̶U̶S̶D̶&quot;
+    	&emsp;&emsp; ̶}̶
+		 ̶}̶"]
+	end
+
+	Router
+	Database[("&emsp;&emsp;&emsp;")]
+
+    Router --> QueryResponse
```

Comment:
Maybe it's just me, but the direction of the arrow from router to `POST` request confuses me here. Wouldn't the `POST` request be made _to_ the router, which propagates it invalidate the cached price info?

<img width="267" alt="image" src="https://github.com/user-attachments/assets/7247e96b-7993-4d8d-bdf0-01978aadc2a8">

Reply:
Oops, this is a copy-paste mistake. Thank you for catching it!